pt-BR/form_helpers.md

```diff
@@ -343,181 +343,180 @@ When parsing POSTed data, Rails will take into account the special `_method` par
 
 IMPORTANT: All forms using `form_with` implement `remote: true` by default. These forms will submit data using an XHR (Ajax) request. To disable this include `local: true`. To dive deeper see [Working with JavaScript in Rails](working_with_javascript_in_rails.html#remote-elements) guide.
 
-Making Select Boxes with Ease
+Fazendo (*Select boxes*) caixas de seleção com facilidade
 -----------------------------
 
-Select boxes in HTML require a significant amount of markup (one `OPTION` element for each option to choose from), therefore it makes the most sense for them to be dynamically generated.
-
-Here is what the markup might look like:
+As caixas de seleção em HTML requerem uma quantidade significativa de marcação (um elemento `OPTION` para cada opção de escolha), portanto, faz mais sentido que sejam geradas dinamicamente. 
 
+Esta é a aparência da marcação:
 ```html
 <select name="city_id" id="city_id">
-  <option value="1">Lisbon</option>
-  <option value="2">Madrid</option>
-  <option value="3">Berlin</option>
+  <option value="1">Lisboa</option>
+  <option value="2">Madri</option>
+  <option value="3">Berlim</option>
 </select>
 ```
 
-Here you have a list of cities whose names are presented to the user. Internally the application only wants to handle their IDs so they are used as the options' value attribute. Let's see how Rails can help out here.
+Aqui você tem uma lista de cidades cujos nomes são apresentados ao usuário. Internamente, o aplicativo deseja apenas manipular seus IDs, para que sejam usados como o atributo de valor das opções. Vamos ver como o Rails pode ajudar aqui.
 
-### The Select and Option Tags
+### As tags de seleção (*Select*) e opção (*Option*)
 
-The most generic helper is `select_tag`, which - as the name implies - simply generates the `SELECT` tag that encapsulates an options string:
+O auxiliar mais genérico é `select_tag`, que - como o nome indica - simplesmente gera a tag `SELECT` que encapsula uma string de opções:
```

Comment:
```suggestion
O auxiliar mais genérico é `select_tag`, que - como o nome indica - simplesmente gera a tag `SELECT` que encapsula uma *string* de opções:
```

Acho que string poderia ficar em itálico por ser termo "estrangeiro" o que acha?